File: List/Node.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace List_DSA
{
    class Node
    {
        public Node Next { set; get; }
        public Object Data { set; get; }

        public Node()
        {
            Next = null;
        }

        public override string ToString()
        {
            return Data.ToString() + " -> " +
                (Next == null ? "NULL" : Next.ToString());
        }
    }
}

File: Treap/Node.java

public class Node {

	public int Data;
	public Node Left;
	public Node Right;
	
	public Node(int data)
	{
		Data = data;
		Left = null;
		Right = null;
	}
	
	public String toString()
	{
		return "[" + (Left != null ? Left.toString() : "END") + " <- " + Data + " -> " +  (Right != null ? Right.toString() : "END") +"]";
	}
}